torch/distributed/_tensor/debug/comm_mode.py

```diff
@@ -56,7 +56,14 @@
     c10d_ops.reduce_scatter_tensor_coalesced_,
 }
 
-trivial_ops = {"aten.detach.default", "aten.t.default", "aten.view.default"}
+trivial_ops = {
+    "aten.detach.default",
+    "aten.t.default",
+    "aten.view.default",
+    "aten._to_copy.default",
+    "aten.as_strided.default",
+    "aten.transpose.int",
```

Comment:
nit: I think we want to preserve `t`, `view` and `transpose` op? these operations might change the sharding, so it would be better user can see how the sharding got changed